GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,37 @@
+# get_repo_json.py
+from time import sleep
+def delayserver():
+    sleep(0.01)
+    return
```

Comment:
```suggestion
    return
```

This line is unneeded.